src/ChainSafe.Gaming.NetCore/NetCoreOperatingSystemMediator.cs

```diff
@@ -4,11 +4,9 @@ namespace ChainSafe.Gaming.NetCore
 {
     public class NetCoreOperatingSystemMediator : IOperatingSystemMediator
     {
-        public string ClipboardContent
-        {
-            get => throw new System.NotImplementedException();
-            set => throw new System.NotImplementedException();
-        }
+        public bool IsMobilePlatform => throw new System.NotImplementedException();
+
+        public Platform Platform => throw new System.NotImplementedException();
```

Comment:
Don't forget to implement those.

Reply:
These aren't referenced or relevant in our current context, they're new properties I've created. They can be implemented when they become relevant.

Reply:
it's inconsequential and out of scope

Reply:
It will break tests probably

Reply:
yeah should I create an issue for it or just leave a //todo?